cmd/sponged/handlers/data.go

```diff
@@ -26,9 +26,9 @@ const defaultVersion = 1
 
 //==============================================================================
 
-// Upsert receives POSTed data, itemizes it then Upserts it via the item service
+// Import receives POSTed data, itemizes it then imports it via the item API.
 // 204 SuccessNoContent, 400 Bad Request, 404 Not Found, 500 Internal.
-func (dataHandle) Upsert(c *app.Context) error {
+func (dataHandle) Import(c *app.Context) error {
 
 	// Unmarshall the data packet from the Request Body.
 	var dat map[string]interface{}
```

Comment:
This is a nil map and then on 43 you are assigning it. This is not necessary since the Data field will be set to its zero value by default. From what I can see the dat variable is not used again?